_data/sites.json

```
@@ -10579,6 +10579,16 @@
         ]
     },
 
+    {
+    	"name": "SimplePlanes",
+    	"url": "https://www.simpleplanes.com/Account/CloseAccount",
+    	"difficulty": "easy",
+    	"notes": "Login to your account, open your profile page and follow these steps: click 'Manage Account' > click 'Close Account' > click 'OK' > type 'CLOSE' > click 'CLOSE' > click 'OK'. Using the link from here also skips every step before \"type 'CLOSE'\".",
+    	"domains": [
+    		"simpleplanes.com"
+    	]
+    },
+
```

```suggestion
    {
        "name": "SimplePlanes",
        "url": "https://www.simpleplanes.com/Account/CloseAccount",
        "difficulty": "easy",
        "notes": "Login to your account, open your profile page and follow these steps: click 'Manage Account' > click 'Close Account' > click 'OK' > type 'CLOSE' > click 'CLOSE' > click 'OK'. Using the link from here also skips every step before \"type 'CLOSE'\".",
        "domains": [
            "simpleplanes.com"
        ]
    },

```

Use whitespaces over tabs